merkle-tree/src/mmcs.rs

```diff
@@ -191,6 +238,7 @@ where
             }
         }
 
+        // Compare the computed root with the committed one.
```

Comment:
Nit: Maybe more informative:
```suggestion
        // The computed root should equal the committed one.
```